lib/ghtorrent/ghtorrent.rb

```diff
@@ -54,6 +55,25 @@ def persister
       @persister
     end
 
+    def load_orgs_file(path)
+      if not File.exists?(path)
+        return nil
+      end
+
+      result = Set.new
+      IO.foreach(path) do |x|
+        x = x.strip
+        if x.empty? == false
+          result.add(x)
+        end
+      end
+      result
+    end
+
+    def includeOrg?(org)
+      return (not @orgFilter.nil?) || (@org_filter.include? org)
```

Comment:
The naming here was bogs as was some of the logic.  There is a revised version coming.